draft-packages/modal/KaizenDraft/Modal/Presets/InformationModal.tsx

```diff
@@ -25,6 +32,7 @@ export type InformationModalProps = Readonly<
     renderBackground?: () => React.ReactNode
     image?: React.ReactNode
     children: React.ReactNode
+    notification?: InlineNotificationProps
```

Comment:
Something like `contentHeader?: children` would be preferred so it's a little more generic.

Reply:
yes will do that! Thanks 🙏